starfish/intensity_table/intensity_table.py

```diff
@@ -146,6 +155,10 @@ def from_spot_data(
             number of dimensions. If this argument is omitted, dimension names
             are taken from ``coords`` (if possible) and otherwise default to
             ``['dim_0', ... 'dim_n']``.
+        ch_values : Sequence[int]
+            The possible values for the channel number, in sequential order.
```

Comment:
what does "sequential order" mean? Does it mean they need to be numerically sorted, or match the input? I feel like we require the latter but not the former.

Reply:
You're right.

Reply:
Fixed.